langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockCohereEmbeddingModel.java

```diff
@@ -0,0 +1,180 @@
+package dev.langchain4j.model.bedrock;
+
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.bedrock.internal.AbstractBedrockEmbeddingModel;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Bedrock Cohere embedding model with support for both versions:
+ * {@code cohere.embed-english-v3} and {@code cohere.embed-multilingual-v3}
+ * <br>
+ * See more details <a href="https://docs.cohere.com/v2/docs/amazon-bedrock">here</a> and
+ * <a href="https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-embed.html">here</a>.
+ */
+public class BedrockCohereEmbeddingModel extends AbstractBedrockEmbeddingModel<BedrockCohereEmbeddingResponse> {
+
+    private String model;
```

Comment:
```suggestion
    private final String model;
```

nit: all properties can be made `final`